components/TransactionDistributionList.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div class="transactionDistributionList">
+    <div
+      class="transactionDistributionList__head transactionDistributionList__box"
```

Comment:
Вероятно будет правильнее 
```suggestion
      class="transactionDistributionList__box_head transactionDistributionList__box"
```
css для __head я не увидел в этом компоненте